apps/wpcom-block-editor/src/wpcom/features/live-preview.tsx

```diff
@@ -0,0 +1,103 @@
+import { useDispatch, useSelect } from '@wordpress/data';
+import domReady from '@wordpress/dom-ready';
+import { __, sprintf } from '@wordpress/i18n';
+import { registerPlugin } from '@wordpress/plugins';
+import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/private-apis';
+import { getQueryArg } from '@wordpress/url';
+import { useEffect } from 'react';
+
+/**
+ * Return true if the user is currently previewing a theme.
+ * FIXME: This is copied from Gutenberg, and that should be accessed from the `core/edit-site` store.
```

Comment:
Suggestion to refine the documentation

```suggestion
 * FIXME: This is copied from Gutenberg; we should be creating a selector for the `core/edit-site` store.
```